CMakeLists.txt

```diff
@@ -12,6 +12,9 @@ include(Options)
 set_options()
 set_directory_options()
 
+include(Static)
+set_static()
```

Comment:
I don't like the name as it is misleading for every platform but MinGW. Either use a better name or also guard it with an ```if (MINGW)``` here so it becomes obvious that it does nothing otherwise.